src/helpers/role/inventory-helper.js

```diff
@@ -8,6 +9,8 @@ export const getInventoryGroups = async ({ name, perPage, page, options } = {})
   return await inventoryResourceTypesApi.apiResourceTypeGetResourceTypeGroupsList(name, perPage, page, options);
 };
 
+export const enableWorkspacesNameChange = useFlag('platform.rbac.groups-to-workspaces-rename');
```

Comment:
Hooks should be called only in function components https://react.dev/reference/rules/rules-of-hooks